src/Service/Resolvers/Sql Query Structures/SqlUpsertQueryStructure.cs

```diff
@@ -72,7 +72,7 @@ public class SqlUpsertQueryStructure : BaseSqlQueryStructure
               authorizationResolver: authorizationResolver,
               gQLFilterParser: gQLFilterParser,
               entityName: entityName,
-              operationType: Config.Operation.Update,
+              operationType: Config.Operation.Upsert,
```

Comment:
Do we have tests that would detect if there's some incorrect behavior happening due to this setting - seems like a bug to me that got introduced in the recent refactoring.

Reply:
So, the action that we are passing here was only used to add the database policy. Because before this PR, we only supported database policy for update action (and not insert action) , we only passed Update. Now we support both create/update.